cpp/ps0000/ps0000_sol.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream> // 2/2
+using namespace std;
+int main() // 1/1
+{
+  cout<<"Hello, World!"<<endl; // 2/4
```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [ ] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line